tests/baselines/reference/capturedParametersInInitializers2.js

```diff
@@ -19,11 +19,13 @@ function foo(y, x) {
     var _a;
 }
 function foo2(y, x) {
-    if (y === void 0) { y = /** @class */ (function () {
-        function class_2() {
-            this[x] = x;
-        }
-        return class_2;
-    }()); }
+    if (y === void 0) { y = (_a = /** @class */ (function () {
+            function class_2() {
+                this[_b] = x;

```

Comment:
I suppose this means that computed key names are evaluated only once, at declaration time, instead of at construction time?

Reply:
Correct, just like computed names for methods already are. We were inconsistent here.